iaso/urls.py

```diff
@@ -261,6 +261,7 @@ def append_datasources_subresource(viewset, resource_name, urlpatterns):
     path("storages/<str:storage_type>/<str:storage_customer_chosen_id>/logs", logs_per_device),
     path("workflows/export/<workflow_id>/", export_workflow, name="export_workflow"),
     path("workflows/import/", import_workflow, name="import_workflow"),
+    path("fhir/", include("iaso.api.fhir.urls")),
```

Comment:
The way you implemented that makes the route unavailable in the default list. Is this done on purpose?

![a](https://github.com/user-attachments/assets/96ddfb98-14f2-421c-87c8-5c8b8bc8e6eb)